web/libs/datamanager/src/components/Common/Table/utils.js

```diff
@@ -7,15 +7,17 @@ export const prepareColumns = (columns, hidden) => {
 
 export const getProperty = (object, path) => {
   try {
-    const normalizedPath = path
-      .split(".")
-      .map((p) => `["${p}"]`)
-      .join("");
-
-    // eslint-disable-next-line no-new-func
-    const fn = new Function("object", `return object${normalizedPath}`);
+    const properties = path.split(".");
+    let result = object;
+
+    for (const property of properties) {
+      result = result?.[property];
+      if (result === undefined) {
+        return undefined;
+      }
```

Comment:
bit of a nit-pick but technically this'll do too
```suggestion
      if (!result) {
        return result;
      }
```